src/browser/modules/D3Visualization/components/DetailsPane.tsx

```diff
@@ -39,17 +40,54 @@ import { StyleableRelType } from './StyleableRelType'
 import { upperFirst } from 'services/utils'
 import { ShowMoreOrAll } from 'browser-components/ShowMoreOrAll/ShowMoreOrAll'
 
+export const MAX_LENGTH_SMALL = 150
+export const MAX_LENGTH_LARGE = 300
+type ExpandableValueProps = {
+  value: string
+  type: string
```

Comment:
This prop seems unused

Reply:
yes, you are right, it is from the future, is going to be used in my next PR :D. Make sense to remove it for now though. 👍